src/renderer/src/components/Messages/MessagesSlice.ts

```
@@ -65,6 +67,23 @@ const initialState = messagesAdapter.getInitialState({
 });
 
 // thunk functions
+export const fetchMessage = createAsyncThunk(
+  'messages/fetchMessage',
+  async (query) => {
+    const response = await IpcService.send<{ message: any }>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)